src/octokit.net.Extensions/GitHubResilientDelegatingHandler.cs

```diff
@@ -52,74 +52,8 @@ public GitHubResilientDelegatingHandler(IAsyncPolicy policy,ILogger logger=null)
 
             _logger?.LogInformation("Response Recieved. Status Code: {statusCode}",httpResponse.StatusCode.ToString());
 
-            var githubRespone = await BuildResponse(httpResponse);
-
-            _logger?.LogInformation("Remaining Limit: {remaining} - Reset At: {reset}",
-                githubRespone.ApiInfo.RateLimit.Remaining,
-                githubRespone.ApiInfo.RateLimit.Reset.ToLocalTime());
-
-            MethodInfo handleErrors = typeof(Connection)
-                .GetMethod("HandleErrors",
-                BindingFlags.NonPublic | BindingFlags.Static);
-
-            try
-            {
-                handleErrors.Invoke(this, new object[] { githubRespone });
```

Comment:
Here I'm calling _HandleErrors_ to throw a potential GitHub exception. In fact, what I need is to have something to throw a GitHub exception, given a valid GitHub response (HttpResponseMessage).